python/semantic_kernel/template_engine/blocks/code_block.py

```diff
@@ -154,7 +154,7 @@ def _enrich_function_arguments(
         return arguments
 
     def _get_function_from_plugin_collection(
```

Comment:
Do we need this method now? If so, should we rename since we removed the "plugin collection" nomenclature?